src/activetasks.js

```diff
@@ -0,0 +1,129 @@
+import nmo from './nmo.js';
+import Wreck from 'wreck';
+import Promise from 'bluebird';
+import log from 'npmlog';
+import prettyjson from 'prettyjson';
+import { validUrl, removeUsernamePw } from './utils';
+
+/* TODO:
+  * active tasks implementation
+  * tests
+  * docs
+*/
+
+export function cli (cluster, filter) {
+  return new Promise((resolve, reject) => {
+    if (!cluster) {
+      const msg = [
+        'Usage:',
+        '',
+        'nmo activetasks <cluster> [<filter>] [--json]',
+        'nmo activetasks <url> [<filter>] [--json]'
+      ].join('\n');
+
+      const err = new Error(msg);
+      err.type = 'EUSAGE';
+      return reject(err);
+    }
+
+    // `cluster` argument can be a cluster or a url.
+    // This function checks that and returns an appropriate url
+    const url = getUrlFromCluster(cluster);
+    getActiveTask(url, filter).then(tasks => {
+      const jsonOut = nmo.config.get('json');
+
+      if (jsonOut) {
+        console.log(tasks);
```

Comment:
I also thought of that. But  I cannot because I need to resolve the promise at the end.